.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us squash bugs!
+labels: bug
+---
+<!--
+
+Please give us an overview of what went wrong, and what you expected to have
+happen instead.
+
+-->
+
+**Steps to reproduce**
+
+<!--
+
+Please provide step-by-step instructions as to how to reproduce the problem.
+
+-->
+
+**When can we consider this to be "fixed"?**
```

Comment:
I'm trying to be somewhat pedantic with the English language here - for better or worse.

This question starting with "when" == "at one time" for me when I first read it. The description clarifies it well, but maybe we should be more to the point.

Maybe:
```suggestion
**Describe how a resolution looks like**
```

Reply:
Is the comment below the heading not clear enough as to what the heading means?